module/VuFind/src/VuFind/Log/Handler/VerbosityTrait.php

```diff
@@ -11,31 +11,26 @@
  * it under the terms of the GNU General Public License version 2,
  * as published by the Free Software Foundation.
  *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
```

Comment:
Was this clause accidentally removed? I think we're supposed to have a consistent license header on every file...